src/file.js

```diff
@@ -24,22 +24,8 @@ Sk.builtin.file = function (name, mode, buffering) {
     } else if (this.name === "/dev/stderr") {
         this.fileno = 2;
     } else {
-        if (Sk.inBrowser) {  // todo:  Maybe provide a replaceable function for non-import files
```

Comment:
Well in the real world you can't have two files with the same path, neither should you really with skulpt. And since there's no write function, you can't actually create a clash between files as a user. 

It might even prove interesting to be able it `import yadayada` where that is a bit of python on the site, in a textarea.